src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -243,21 +243,106 @@ public Rule TransformRuleVersionOne(RuleVersionOne node)
             return rule;
         }
 
+        private List<FileLocation> CreateFileLocationsFromDictionary(IDictionary<string, string> dictionary, Run run)
+        {
+            List<FileLocation> result = null;
+
+            if (dictionary != null)
+            {
+                result = new List<FileLocation>();
```

Comment:
Name this `fileLocations` -- a better name in general, since "result" is so generic, but especially so in this context, where `result` has a very different meaning!

Reply:
Yep meant to do that...